e2e/__tests__/__snapshots__/testFailingJasmine.test.ts.snap

```diff
@@ -34,6 +34,21 @@ FAIL __tests__/worksWithConcurrentMode.test.js
 
       at Object.toBe (__tests__/worksWithConcurrentMode.test.js:10:16)
 
+  ● block with concurrent › encountered a declaration exception
+
+    TypeError: test.concurrent.failing.each is not a function
```

Comment:
we should end up with ``Jest: \`failing\` tests are only supported in \`jest-circus\`.`` like below

Reply:
Will fix this and push the changes, thanks!